Earthfile

```diff
@@ -60,7 +60,7 @@ integration-test-base:
         apk del .build-dependencies && rm -f msodbcsql*.sig mssql-tools*.apk
     ENV PATH="/opt/mssql-tools/bin:${PATH}"
 
-    GIT CLONE https://github.com/elixir-ecto/ecto_sql.git /src/ecto_sql
+    GIT CLONE --branch composite_foreign_keys https://github.com/soundmonster/ecto_sql.git /src/ecto_sql
```

Comment:
This branch needs a few changes to the base migration used for the integration tests; I've opened a [PR](https://github.com/elixir-ecto/ecto_sql/pull/324) for this so I hope to be able to drop this line soon.